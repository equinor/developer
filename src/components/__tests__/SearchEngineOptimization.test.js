import React from "react";
import renderer from "react-test-renderer";
import { StaticQuery } from "gatsby";
import SearchEngineOptimization from "../SearchEngineOptimization";

beforeEach(() => {
  StaticQuery.mockImplementationOnce(({ render }) =>
    render({
      site: {
        siteMetadata: {
          title: `Default Starter`,
        },
      },
    })
  );
});

describe("SearchEngineOptimization", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<SearchEngineOptimization title="All blogs" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("renders correctly with keywords", () => {
    const tree = renderer
      .create(
        <SearchEngineOptimization title="All blogs" keywords={["blog"]} />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
