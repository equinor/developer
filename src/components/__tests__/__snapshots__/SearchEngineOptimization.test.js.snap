// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchEngineOptimization renders correctly 1`] = `null`;

exports[`SearchEngineOptimization renders correctly with keywords 1`] = `null`;
